packages/realtime/src/subscribe/TokenSubscription.ts

```diff
@@ -71,8 +71,12 @@ export class TokenSubscription {
 
     if (this.#app.apiBaseUrl) {
       url = new URL(path, this.#app.apiBaseUrl);
+
+      // If INNGEST_API_BASE_URL=url is set instead of INNGEST_DEV=url, assume
+      // that we should be connecting to a dev server
+      url.protocol = "ws:";
```

Comment:
Instead of inferring whether TLS is enabled for the WebSocket URL, should we use `url.protocol`'s current value and use `ws` or `wss` depending on whether it's http or https?

Reply:
Brain fart! 💯